<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magic Album</string>
  </property>
  <property name="windowOpacity">
   <double>0.999000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="top" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="FlatPushButton" name="iconBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titlelbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Californian FB&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>455</width>
          <height>52</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="FlatPushButton" name="minBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FlatPushButton" name="maxBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FlatPushButton" name="exitBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="middle" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="FlatCtrlButton" name="leftBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="center" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" rowspan="2" colspan="2">
          <widget class="QLabel" name="movielbl">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Hello        World</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="FlatCtrlButton" name="rgtBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="statuslbl">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Cambria Math&quot;;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>654</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="FlatPushButton" name="fullBtn" native="true">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlatCtrlButton</class>
   <extends>QWidget</extends>
   <header location="global">flatctrlbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlatPushButton</class>
   <extends>QWidget</extends>
   <header location="global">flatpushbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
